modules/importer/src/endpoints.rs

```diff
@@ -281,6 +281,10 @@ async fn delete(
 #[utoipa::path(
     tag = "importer",
     operation_id = "listImporterReports",
+    params(
+        Query,
+        Paginated,
+    ),
```

Comment:
Just out of curiosity why is it that this block of code does not need something like:
```
params(
        ("name", Path, description = "Name of the Importer"),
        Query,
        Paginated,
    ),
```

I saw similar endpoints like https://github.com/trustification/trustify/blob/main/modules/fundamental/src/sbom/endpoints/mod.rs#L323 

and those endpoints use `("id", Path, description = "ID of the SBOM to get packages for"),` but not this endpoint.

I am not saying there is anything wrong, I actually tested the openapi.yaml generated by this PR and it worked like a charm, just asking to fix my ignorance on these matters :)

Reply:
I honestly don't know. I just copied it from another working endpoint. :smile:

Reply:
Might be because the `name` is contained within the `#[get("/v2/importer/{name}/report")]` macro already, so the endpoint wouldn't even be resolved without it? Not sure. But since it seems to be working, I'm gonna merge.